draft-ietf-trans-rfc6962-bis.md

```diff
@@ -581,10 +582,12 @@ The consistency proof between hash2 and hash is PROOF(6, D\[7]) = \[i, j, k].
 k, i are used to verify hash2, and j is additionally used to show hash is
 consistent with hash2.
 
-### Signatures    {#signatures}
+## Signatures    {#signatures}
 
 Various data structures are signed. A log MUST use one of the signature
-algorithms defined in {{signature_algorithms}}.
+algorithms defined in {{signature_algorithms}}. Data structure are encoded
+according to Section 4 of [RFC5246] for the purpose of signing and signature
+validation.
```

Comment:
We already reference RFC5246 Section 4 in the "Data Structures" section (currently numbered 1.2).

How about changing section 1.2 to say...
`Data structures are defined **and encoded** according to the conventions laid out in Section 4 of [RFC5246].`

...and then change this "Signatures" section to reference section 1.2...
`Various data structures {{data-structures}} are signed. A log MUST use one of the signature algorithms defined in {{signature_algorithms}}.`